File: mobile/android/air/SubModules/UISettings/src/main/java/app/twallet/air/uisettings/viewControllers/settings/models/SettingsSection.kt
package app.twallet.air.uisettings.viewControllers.settings.models

data class SettingsSection(
    val section: Section,
    val title: String,
    var children: List<SettingsItem>
) {
    enum class Section {
        ACCOUNTS,
        PORTFOLIO,
        SETTINGS,
    }
}

File: mobile/ios/Air/SubModules/UISettings/Settings/Models/SettingsSection.swift
//
//  SettingsSection.swift
//  UISettings
//
//  Created by Sina on 6/26/24.
//

import Foundation

struct SettingsSection: Identifiable {
    enum Section: String, CaseIterable, Equatable, Hashable {
        case header
        case accounts
        case tabsAndModules
        case general
    }
    let id: Section
    var children: [SettingsItem]
}
